examples/src/main/scala/org/apache/spark/examples/SparkTC.scala

```diff
@@ -55,20 +55,24 @@ object SparkTC {
     // the graph to obtain the path (x, z).
 
     // Because join() joins on keys, the edges are stored in reversed order.
-    val edges = tc.map(x => (x._2, x._1))
+    val edges = tc.map(x => (x._2, x._1)).cache()
```

Comment:
This is more complex for an example, but, I think it's reasonably important to show this pattern in this type of example.